### DISCUSSION TITLE
Getting query parameters in Android for version >= v10

### DISCUSSION THREAD

**Discussion by timhooker** [react-native-webview/react-native-webview#2505] [Q&A]
Our mobile app has an interaction with a web based form that utilizes rails cookies to track the user. After the form is submitted, it does a number of cross domain redirects and returns a code as part of a query string. i.e. `succcess?code=foobar`

Quite suddenly within the last week, for android only, the query parameters stopped being returned for all requests from onNavigationStateChange but also from all other methods. If I downgrade react-native-webbview to 9.4.0 I can receive the query parameters.  ( v10.10.2 is also unsuccessful ) 

I have tried various settings like 
- enableThirdPartyCookies
- sharedCookiesEnabled
- mixedContentMode: 'always'
- and forcing a useragent string from a safari ( since iOS works ) but no dice.

Does this ring any bells with anyone as to what could be going wrong? I thought I'd ask here before opening an issue.

**Comment by jpvdplasse** [comment]
Hi @timhooker , did you manage to fix this? 
I do not get query parameters in both android and iOS with the latest version..

**Comment by kamelmojawaz1** [comment]
@timhooker @jpvdplasse Any luck with this? I have the same issue, it works on iOS but not android?

**Reply by timhooker** [reply]
For a period of time we gated the version but this doesn't seem to be a problem any longer.
